### ISSUE TITLE
[SDK] Getting a GraphQL error when trying to retrieve history for an issue

### ISSUE THREAD

**Issue by ckastbjerg** [linear/linear#91]
Hi 👋 

Thanks for building an awesome issue tracking tool!

I've been playing around with the Linear SDK with the purpose of creating a custom stand-up/daily tool for my team. While doing this, I bumped into a challenge when trying to retrieve the history for an issue.

A bit of a contrived example here:

```ts
const issue = await client.issue("some-issue-id")
const history = await issue.history()
```

This gives me the error:

```
'Field "relationChanges" of type "[IssueRelationHistoryPayload!]" must have a selection of subfields. Did you mean "relationChanges { ... }"?'
```

I don't get the error for similar queries, e.g.

```ts
const labels = await issue.labels() // works just fine
```

Is this a bug? Or am I just doing something wrong?

> 💁 Note, I'm pretty new to both the SDK and GraphQL

**Complete stack trace**:

```
Ti [Error]: Field "relationChanges" of type "[IssueRelationHistoryPayload!]" must have a selection of subfields. Did you mean "relationChanges { ... }"?: {"response":{"errors":[{"message":"Field \"relationChanges\" of type \"[IssueRelationHistoryPayload!]\" must have a selection of subfields. Did you mean \"relationChanges { ... }\"?","locations":[{"line":26,"column":3}],"extensions":{"type":"graphql error","userError":false}}],"status":400,"headers":{}},"request":{"query":"query issue_history($id: String!, $after: String, $before: String, $first: Int, $includeArchived: Boolean, $last: Int, $orderBy: PaginationOrderBy) {
      issue(id: $id) {
        history(
          after: $after
      before: $before
      first: $first
      includeArchived: $includeArchived
      last: $last
      orderBy: $orderBy
    ) {
          ...IssueHistoryConnection
    }
  }
}

fragment IssueHistoryConnection on IssueHistoryConnection {
      nodes {
        ...IssueHistory
  }
  pageInfo {
        ...PageInfo
  }
}

fragment IssueHistory on IssueHistory {
      relationChanges
  addedLabelIds
  removedLabelIds
  source
  issue {
        id
  }
  updatedAt
  toCycle {
        id
  }
  toParent {
        id
  }
  toProject {
        id
  }
  toState {
        id
  }
  fromCycle {
        id
  }
  fromParent {
        id
  }
  fromProject {
        id
  }
  fromState {
        id
  }
  fromTeam {
        id
  }
  toTeam {
        id
  }
  archivedAt
  createdAt
  id
  fromAssignee {
        id
  }
  toAssignee {
        id
  }
  actor {
        id
  }
  fromDueDate
  toDueDate
  fromEstimate
  toEstimate
  fromPriority
  toPriority
  fromTitle
  toTitle
  archived
  updatedDescription
  autoArchived
  autoClosed
}

fragment PageInfo on PageInfo {
      startCursor
  endCursor
  hasPreviousPage
  hasNextPage
}
","variables":{"id":"ISSUE_ID_WAS_HERE"}}}
      at Ii.<anonymous> (/Users/christian/code/linear-stand-up/node_modules/@linear/sdk/dist/index-cjs.min.js:17:28196)
      at Generator.next (<anonymous>)
      at l (/Users/christian/code/linear-stand-up/node_modules/@linear/sdk/dist/index-cjs.min.js:16:416)
      at runMicrotasks (<anonymous>)
 ```

**Comment by ckastbjerg** [comment]
FYI, I switched to using the GraphQL API instead. 

And it's now become clear to me, that the problemt is that the query has `relationChanges` but not `relationChanges { identifier }` in the SDK.

**Comment by eldh** [comment]
Thanks for reporting. We'll try to get a release out today with a fix.

**Comment by eldh** [comment]
Should be fixed in `v1.11.0`
